app/components/CalendarIcon.tsx

```diff
@@ -0,0 +1,66 @@
+import styled from 'styled-components';
```

Comment:
I was going to refactor the `datePickerWidget` but then decided against it, though I kept this pulled out if anyone is wondering what this is about.